src/openforms/utils/json_logic.py

```diff
@@ -49,3 +49,123 @@ def normalize_value(value: Union[dict, list, tuple, int, str]) -> JSONLogicValue
         return JsonLogicTest.from_expression(value)
 
     raise NotImplementedError(f"Unknown value type: {type(value)}")
+
+
+def find_json_logic_test_value(value: JsonLogicTest) -> str:
+
+    """returns the value inside a JsonLogicTest"""
+    if isinstance(value, JsonLogicTest) and value.operator == "date":
+        return find_json_logic_test_value(value.values[0])
+    if isinstance(value, JsonLogicTest) and value.operator == "var":
+        return value.values[0]
+
+    else:
+        return value
+
+
+def convert_json_logic_in_json_tree(
+    logics, keys_mapped_to_component, keys_mapped_to_form, updated_submission_data
+):
+
+    """returns a dict (potentially nested) with complemetentary data regarding the field"""
```

Comment:
```suggestion
    """returns a dict (potentially nested) with complemetentary data regarding the field"""
```
docstring hygiene :)

I would also very much like to see some examples of the shape of data returned, because this docstring doesn't really tell me anything useful except that it's a dict with some structure returned. It's also unclear what "the field" is in this context.